Tests/Unit/Model/RatepayInvoicePaymentMethodTest.php

```diff
@@ -342,4 +344,39 @@ public function onBeforeOrderCreationFailedProvider()
             ],
         ];
     }
+
+    /**
+     * @dataProvider getPostProcessingTransactionProvider
+     */
+    public function testGetPostProcessingTransaction($aOrderItems)
+    {
+        $oParentTransaction = $this->getMockBuilder(Transaction::class)
+            ->setMethods(['getResponseXML'])
+            ->getMock();
+        $oParentTransaction
+            ->method('getResponseXML')
+            ->willReturn(file_get_contents(__DIR__ . '/../../resources/success_response.xml'));
+
+        $sResult = $this->_oPaymentMethod->getPostProcessingTransaction(Transaction::ACTION_CREDIT, $oParentTransaction, $aOrderItems);
+
+        $this->assertInstanceOf(RatepayInvoiceTransaction::class, $sResult);
```

Comment:
A better check would be that the corresponding items are mapped correctly in the transaction.

`$oResult->mappedProperties()['order-items']`